airbyte-webapp/src/components/ui/DatePicker/DatePicker.module.scss

```
@@ -0,0 +1,147 @@
+/* stylelint-disable selector-class-pattern, no-descending-specificity */
+
+@use "scss/colors";
+@use "scss/fonts";
+@use "scss/variables";
+@use "scss/z-indices";
+
+.wrapper {
+  position: relative;
+}
+
+.datepickerButtonContainer {
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 33px;
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  border: none;
+}
+
+.datepickerButton svg {
+  font-size: 14px;
+}
+
+.popup {
+  z-index: z-indices.$datepicker;
+}
+
+.input {
+  padding-right: 25px;
+}
+
+:global(.react-datepicker) {
+  display: flex;
+  color: colors.$dark-blue-900;
+  font-family: fonts.$primary;
+  border: variables.$border-thick solid colors.$grey-100;
+  border-radius: variables.$border-radius-md;
+  box-shadow: 2px 2px 20px -6px colors.$grey-200;
+}
+
+/** Main calendar area **/
+:global(.react-datepicker__header) {
+  background-color: colors.$grey-50;
+  border-top-right-radius: variables.$border-radius-md;
+  border-top-left-radius: variables.$border-radius-md;
+  border-bottom: none;
+}
+
+:global(.react-datepicker__header.react-datepicker__header--has-time-select) {
+  border-top-right-radius: 0;
+}
+
+:global(.react-datepicker__header:not(.react-datepicker__header--has-time-select)) {
+  border-top-right-radius: variables.$border-radius-md;
+}
+
+:global(.react-datepicker__day-name) {
+  color: colors.$grey-300;
+}
+
+:global(.react-datepicker__current-month) {
+  color: colors.$dark-blue-900;
+  font-weight: 500;
+  margin-bottom: variables.$spacing-md;
+}
+
+:global(.react-datepicker__day) {
+  color: colors.$dark-blue-900;
+  background-color: transparent;
+  border-radius: variables.$border-radius-xs;
+
+  &:hover {
+    background-color: colors.$grey-100;
+  }
+}
+
+:global(.react-datepicker__day--outside-month) {
+  color: colors.$grey-300;
+}
+
+:global(.react-datepicker__day--today) {
+  background-color: colors.$grey-50;
+  font-weight: 7 00;
```

```suggestion
  font-weight: 700;
```
I think this was a typo?

---

Yes, definitely 🙂